packages/reports/src/Containers/profit-table.jsx

```diff
@@ -16,31 +16,22 @@ import PlaceholderComponent from '../Components/placeholder-component.jsx';
 import { ReportsMeta } from '../Components/reports-meta.jsx';
 import { getProfitTableColumnsTemplate } from 'Constants/data-table-constants';
 
-const profit_tablews_href = urlFor('user/profit_tablews', { legacy: true });
-
-const getRowAction = row_obj =>
-    getSupportedContracts()[extractInfoFromShortcode(row_obj.shortcode).category.toUpperCase()] &&
-    !isForwardStarting(row_obj.shortcode, row_obj.purchase_time_unix)
+const getRowAction = row_obj => {
+    const unsupported_contract_type = extractInfoFromShortcode(row_obj.shortcode).category.toUpperCase();
+    return getSupportedContracts()[unsupported_contract_type] &&
```

Comment:
This is not needed and why is this called unsupported? shouldn't be the supported contracts?